app/src/pages/inside/historyPage/historyView/historyTable/emptyHistoryItem/emptyHistoryItem.jsx

```diff
@@ -67,10 +80,15 @@ export class EmptyHistoryItem extends Component {
   };
 
   render() {
+    const {
+      intl: { formatMessage },
+    } = this.props;
     return (
-      <div className={cx('empty-history-item')}>
-        <div className={cx('item-text-content')}>{this.renderTextContent()}</div>
-      </div>
+      <EmptyHistoryItemTooltip tooltipContent={formatMessage(messages.emptyItemTooltip)}>
+        <div className={cx('empty-history-item')}>
+          <div className={cx('item-text-content')}>{this.renderTextContent()}</div>
+        </div>
+      </EmptyHistoryItemTooltip>
```

Comment:
You have added a tooltip for all content, but it is only required for item without executions (for [this](https://github.com/reportportal/service-ui/pull/2613/files#diff-f04b75348bbbd0050f74907b995ff1690087e60586b776f264379ecf00e23cc9R67) line)